src/styles/mixins/buttons.scss

```diff
@@ -133,3 +133,15 @@
     text-decoration: underline;
   }
 }
+
```

Comment:
I think a comment here could be helpful (like the button--link mixin above) so its clear where it should be used.

Reply:
Great feedback!